website/docs/programming-languages.md

```diff
@@ -42,3 +42,4 @@ For an actual example of an issue or pull request adding the above support, plea
 |   PHP    |                    🚫                    |                       🚫                        |
 |   Perl   |                    🚫                    |                       🚫                        |
 |  Scala   |                    🚫                    |                       🚫                        |
+|  Kotlin  |                    🚫                    |                       🚫                        |
```

Comment:
Could you add a row in line 31 to indicate version supporting kotlin?

Reply:
done